receiver/flinkmetricsreceiver/metadata.yaml

```diff
@@ -0,0 +1,522 @@
+name: flinkmetricsreceiver
+
+resource_attributes:
+  host:
+    description: The host, used to distinguish different jobmanager metrics.
+    type: string
+  taskmanager_id:
+    description: The taskmanager ID, used to distinguish taskmanager metrics.
+    type: string
+  job_name:
+    description: The job name, used to distinguish job metrics.
+    type: string
+  task_name:
+    description: The task name, used to distinguish tasks metrics.
+    type: string
+  subtask_index:
+    description: The subtask index, used to distinguish operator metrics.
+    type: string
+
+attributes:
+  operator_name:
+    description: The operator name, used to distinguish operator metrics.
+    type: string
+  garbage_collector_name:
+    description: The garbage collector name.
+    type: string
+  checkpoint:
+    description: The amount of checkpoints of a given type. 
```

Comment:
The "amount" part is presumably what a metric measures. Can you revisit this description to describe how the attribute is differentiating between data points in the same metric?

Reply:
simplified definition to `The checkpoint type.`

Reply:
added garbage collector enum since there are only a couple and added some details to the description